viewing_party/party.py

```diff
@@ -0,0 +1,120 @@
+##WAVE 01##
+def create_movie(title,genre,rating):
+    new_movie = {}
+    empty = None
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return empty
```

Comment:
This works great but you could just return the title. genre, and rating without doing new_movie = {} and empty = None

```suggestion
def create_movie(title, genre, rating):
    if title == None or genre == None or rating == None:
        return None
    else:
        return {
        "title":title, 
        "genre":genre, 
        "rating":rating
        }
```